knotx-core/knotx-repository/src/main/java/com/cognifide/knotx/repository/impl/LocalRepository.java

```diff
@@ -61,10 +63,18 @@ static LocalRepository of(String path, String catalogue, FileSystem fileSystem)
         return fileSystem.openObservable(localFilePath, new OpenOptions())
                 .flatMap(AsyncFile::toObservable)
                 .flatMap(buffer -> RepositoryResponse.success(buffer.toString(), MultiMap.caseInsensitiveMultiMap()).toObservable())
-                .defaultIfEmpty(RepositoryResponse.error("No Template found for path <%s>", repositoryRequest.getPath()))
+                .defaultIfEmpty(RepositoryResponse.success("", MultiMap.caseInsensitiveMultiMap()))
```

Comment:
Should be `success` returned when no file of template in local repo found?
Maybe there should be:
`.defaultIfEmpty(RepositoryResponse.error(404, "", MultiMap.caseInsensitiveMultiMap()))` instead?